src/bot/commands/news.ts

```diff
@@ -30,11 +31,16 @@ export default (discordMessage: Message) => {
             return;
         }
     }
+    
+    // Check whether the original message should be removed
+    const direct = !!match(message, directRegex);
 
     // Get just the user comment
     const comment = message
         // Remove mentions
         .replace(/<@\d+>/g, "")
+        // Remove direct command
+        .replace(directRegex, "")
```

Comment:
When using `.replace`, it will replace the entire match, not just the first matching group